src/io/flutter/view/OpenFlutterViewAction.java

```diff
@@ -39,7 +40,11 @@ public void actionPerformed(@NotNull final AnActionEvent e) {
 
     FlutterInitializer.sendAnalyticsAction(this);
 
-    final ToolWindow toolWindow = ToolWindowManager.getInstance(project).getToolWindow("Flutter");
-    toolWindow.show(null);
+    final ToolWindow toolWindow = ToolWindowManager.getInstance(project).getToolWindow(FlutterView.TOOL_WINDOW_ID);
```

Comment:
Looks like getToolWindow() can return null if `FlutterViewCondition` returns false, which will happen if we don't recognize any modules as flutter modules.